### PR TITLE
Doc correct then term of window function

### PR DESCRIPTION
**Pull request by zhongjiajie**
ATT, just some window function term word change

### CODE REVIEW

**Comment by zhongjiajie** [comment]
@martint  Hi, Martin PTAL

File: docs/src/main/sphinx/admin/spill.rst [hunk 1]
```diff
@@ -133,6 +133,6 @@ Window functions
 Window functions perform an operator over a window of rows, and return one value
 for each row. If this window of rows is large, a significant amount of memory may
 be needed. When spill to disk for window functions is enabled, if there is not enough
-memory, intermediate sorted results are written to disk. They are loaded back and
```
Comment (sopel39) [inline docs/src/main/sphinx/admin/spill.rst:136]: technically, they are `sorted`.
`cumulated` is also not that meaningful. Remove it altogether?

Comment (zhongjiajie) [reply docs/src/main/sphinx/admin/spill.rst:136]: @sopel39 Did you think we should remove this sentence?

Comment (zhongjiajie) [reply docs/src/main/sphinx/admin/spill.rst:136]: @sopel39 Did you think we should remove this sentence?

Comment (sopel39) [reply docs/src/main/sphinx/admin/spill.rst:136]: I mean `sorted/cumulated` word itself

Comment (zhongjiajie) [reply docs/src/main/sphinx/admin/spill.rst:136]: @sopel39 Ok, I remove the keyword `sorted/cumulated`, more details in https://github.com/trinodb/trino/pull/8881/commits/bbaed40b88c88a41f9a04786ef22c2f8f3cbeb50

**Review by sopel39 — APPROVED** [review]

**Comment by sopel39** [comment]
please squash commits

**Comment by zhongjiajie** [comment]
> please squash commits

Done, PTAL @sopel39

**Comment by sopel39** [comment]
@zhongjiajie Could you sign CLA (https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf)?

**Comment by zhongjiajie** [comment]
> @zhongjiajie Could you sign CLA (https://github.com/trinodb/cla/raw/master/Trino%20Foundation%20Individual%20CLA.pdf)?

Ok, I forgot about it.

**Comment by zhongjiajie** [comment]
FYI, I had signed my CLA.

**Pull request merged**